src/TogglePassword/doc/index.rst

```diff
@@ -0,0 +1,216 @@
+
+Customizing the design
+~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony UX TogglePassword provides a default stylesheet in order to ease
+usage. You can disable it to add your own design if you wish.
+
+In ``assets/controllers.json``, disable the default stylesheet by
+switching the ``@symfony/ux-toggle-password/dist/style.min.css`` autoimport to
+``false``:
+
+.. code-block:: json
+
+    {
+        "controllers": {
+            "@symfony/ux-toggle-password": {
+                "toggle-password": {
+                    "enabled": true,
+                    "fetch": "eager",
+                    "autoimport": {
+                        "@symfony/ux-toggle-password/dist/style.min.css": false
+                    }
+                }
+            }
+        },
+        "entrypoints": []
+    }
+
+.. note::
+
+   *Note*: you should put the value to ``false`` and not remove the line
+   so that Symfony Flex won't try to add the line again in the future.
+
+Once done, the default stylesheet won't be used anymore and you can
+implement your own CSS on top of the TogglePassword.
+
+You can also only customize specific TogglePassword elements by overriding the default classes.
+Simply add your custom class name(s) using the ``button_classes`` option for the toggle element.
+A ``toggle_container_classes`` option is also available to customize the container form theme element::
+
+    // ...
+    use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+
+    class CredentialFormType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder
+                // ...
+                ->add('password', PasswordType::class, [
+                    'toggle' => true,
+                    'button_classes' => ['btn', 'primary', 'my-custom-class'],
+                    'toggle_container_classes' => ['input-group-text', 'my-custom-container'],
+                ])
+                // ...
+            ;
+        }
+
+        // ...
+    }
+
+Extend the Default Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony UX TogglePassword allows you to extend its default behavior using a
+custom Stimulus controller:
+
+.. code-block:: javascript
+
+    // my-toggle-password_controller.js
+
+    import { Controller } from '@hotwired/stimulus';
+
+    export default class extends Controller {
+        connect() {
+            this.element.addEventListener('toggle-password:connect', this._onConnect);
+            this.element.addEventListener('toggle-password:toggle', this._onToggle);
+        }
+
+        disconnect() {
+            // You should always remove listeners when the controller is disconnected to avoid side-effects
+            this.element.removeEventListener('toggle-password:connect', this._onConnect);
+            this.element.removeEventListener('toggle-password:toggle', this._onToggle);
+        }
+
+        _onConnect(event) {
+            // The TogglePassword was just created
```

Comment:
Show a quick example of the relevant `event.detail` stuff they will have access to